Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -1155,47 +1155,48 @@ internal void MirrorUpdate()
             if (observers != null && observers.Count > 0)
             {
                 // one writer for owner, one for observers
-                NetworkWriter ownerWriter = NetworkWriterPool.GetWriter();
-                NetworkWriter observersWriter = NetworkWriterPool.GetWriter();
-
-                // serialize all the dirty components and send (if any were dirty)
-                OnSerializeAllSafely(false, ownerWriter, out int ownerWritten, observersWriter, out int observersWritten);
-                if (ownerWritten > 0 || observersWritten > 0)
+                using (NetworkWriter ownerWriter = NetworkWriterPool.GetWriter())
                 {
-                    // populate cached UpdateVarsMessage and send
-                    varsMessage.netId = netId;
-
-                    // send ownerWriter to owner
-                    // (only if we serialized anything for owner)
-                    // (only if there is a connection (e.g. if not a monster),
-                    //  and if connection is ready because we use SendToReady
-                    //  below too)
-                    if (ownerWritten > 0)
+                    using (NetworkWriter observersWriter = NetworkWriterPool.GetWriter())
```

Comment:
you can put both writer usings in one line: https://stackoverflow.com/questions/1329739/nested-using-statements-in-c-sharp/1329765#1329765

Reply:
Done